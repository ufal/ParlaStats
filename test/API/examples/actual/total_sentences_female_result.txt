[{'person.sex': 'F', 'total_sentences': 18642}]

Running time 7.7900390625 ms

+------------+-----------------+
| person.sex | total_sentences |
+------------+-----------------+
|     F      |      18642      |
+------------+-----------------+
