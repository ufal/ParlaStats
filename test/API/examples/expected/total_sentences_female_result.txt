[{'person.sex': 'F', 'total_sentences': 18642}]

Running time 7.949951171875 ms

+------------+-----------------+
| person.sex | total_sentences |
+------------+-----------------+
|     F      |      18642      |
+------------+-----------------+
